packages/registry/src/types/Blocks.ts

```diff
@@ -0,0 +1,156 @@
+import { Content } from './Content';
+
+export interface BlocksConfig {
+  [key: string]: unknown;
+  title?: string;
+  description?: number;
+  slate?: SlateBlock;
+}
+
+interface BlockConfigBase {
+  /**
+   * The programmatic id of the block
+   */
+  id: string;
+  /**
+   * // The display name of the block (translatable)
+   */
+  title: string;
+  /**
+   * The icon used in the block chooser
+   */
+  icon: string;
+  /**
+   * The group of the block
+   */
+  group: string;
+  /**
+   * The view mode component
+   */
+  view: React.ComponentType;
+  /**
+   * The edit mode component
+   */
+  edit: React.ComponentType;
+  /**
+   * The group (blocks can be grouped, displayed in the chooser)
+   */
+  schema?: object;
+  /**
+   * The group of the block
+   */
+  blockSchema?: ({ props, intl }: { props: unknown; intl: unknown }) => object;
```

Comment:
Another readability thingy, not needed twice:

`blockSchema?: ({ props, intl }: { props: unknown; intl: unknown }) => object;`

`blockSchema?: (payload: { props: unknown; intl: unknown }) => object;`

also `Record<string, unkown>` instead of `object` seams more explicit to me in this cases.